### PR TITLE
updated instructions to match perfect projects

### PR DESCRIPTION
**Pull request by kaleem99**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by Sbonelo01 — APPROVED** [review]

**Review by FaithMo — APPROVED** [review]

**Review by Vuyisanani — APPROVED** [review]

File: content/projects/oop/person/_index.md [hunk 1]
```diff
@@ -76,11 +76,16 @@ Also, you must submit good clean code! Make sure you name your variables clearly
 
 ## Instructions for reviewer
 
-1. **Multiple interests, commas and spacing** - Make sure that the interests array attribute can take in multiple interests and they all get printed out separated with a comma and space, and the last interest should have an ``and`` before it also your class should contain and use all of the attributes above.
-2. **Single interest or no interests** - Check if the class can work with single interests, and the string output should be different from the multiple interest output instead of `my interests are` it should be `my interest is`. Also, if there are no interests then the output should a string with the first 3 attributes name, age, and male `Hello, my name is Ryan, my gender is male and I am 30 years old.` and you could add something like `I have no interests`.
+1. **Multiple interests, commas, punctuation, and spacing** - Make sure that the interests array attribute can take in multiple interests and they all get printed out separated with a comma and space, and the last interest should have an `and` before it followed by a full stop. Your class should contain and use all of the attributes above.
```
Comment (ry-oc) [inline content/projects/oop/person/_index.md]: Please revise the first sentence here, maybe break it down into more than one sentence, it is long and difficult to read.

**Review by ry-oc — CHANGES_REQUESTED** [review]
1 sentence should be revised to make it easier to read

**Review by Sbonelo01 — APPROVED** [review]

File: content/projects/oop/person/_index.md [hunk 2]
```diff
@@ -76,13 +76,13 @@ Also, you must submit good clean code! Make sure you name your variables clearly
...
+   e.g. 2 or more interests should look like this: `Hello, my name is Ryan, my gender is male and I am 30 years old. My interests are reading and coding.`.
 
 2. **Single interest or no interests** - Check if the class can work with a single interest and that the string output is different from the multiple interests string output. Instead of `my interests are` it should be `my interest is`.
-If there are no interests then the output should be a string with the first 3 attributes - name, age, and male.
-e.g. No interests should look like this: `Hello, my name is Ryan, my gender is male and I am 30 years old.`.
-1 interest should look like this: `Hello, my name is Ryan, my gender is male and I am 30 years old. My interest is being a hardarse.`.
+   If there are no interests then the output should be a string with the first 3 attributes - name, age, and male.
+   e.g. No interests should look like this: `Hello, my name is Ryan, my gender is male and I am 30 years old.`.
+   1 interest should look like this: `Hello, my name is Ryan, my gender is male and I am 30 years old. My interest is being a hardarse.`.
```
Comment (FaithMo) [inline content/projects/oop/person/_index.md]: please remove the indentation here and on line 80

**Review by FaithMo — CHANGES_REQUESTED** [review]

**Review by FaithMo — APPROVED** [review]

**Review by Vuyisanani — APPROVED** [review]

**Review by sheenarbw — CHANGES_REQUESTED** [review]
This doesn't fix the problem

If you look at the automarker config: The python tests and JS tests expect different things. 
1. The project instructions should state exactly what is requiered
2. the configuration should be consistent

Python
```
def test_no_interests():
    hanabi = Person("Hanabi", 31, "female", [])
    option1 = "Hello, my name is Hanabi, my gender is female and I am 31 years old. I have no interests."
    option2 = "Hello, my name is Hanabi, my gender is female and I am 31 years old."
    assert hanabi.hello() in [option1, option2]
``` 

JS
```
  it("should handle no interests and the hello method should include 'I have no interests'", () => {
    const person = new Person("Ryan", 30, "male");
    expect(person.hello()).toBe(
      "Hello, my name is Ryan, my gender is male and I am 30 years old. I have no interests."
    );
  });
```

**Review by Sbonelo01 — APPROVED** [review]

**Review by FaithMo — APPROVED** [review]

**Review by KhalidUmuzi — APPROVED** [review]

**Review by ry-oc — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**